modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,564 @@
+
+Example:
+```bash
+grep data_directory /etc/redpanda/redpanda.yaml
+    data_directory: /var/lib/redpanda/data
+df -khT /var/lib/redpanda/data
+```
+
+Output:
+```bash
+Filesystem     Type  Size  Used Avail Use% Mounted on
+/dev/nvme0n1   xfs   1.8T   14G  1.8T   1% /mnt/vectorized
+```
+
+Output:
+
+```bash
+Filesystem     Type  Size  Used Avail Use% Mounted on
+/dev/md0       xfs    14T   99G   14T   1% /mnt/vectorized
+```
+
+Output:
+```bash
+mdadm: An option must be given to set the mode before a second device (/dev/md0) is listed
+```
+
+Example:
+```bash
+mdadm --detail /dev/md0
+```
+
+Output:
+```bash
+/dev/md0:
+           Version : 1.2
+     Creation Time : Thu Apr 18 11:03:46 2024
+        Raid Level : raid0
+        Array Size : 14648172544 (13969.59 GiB 14999.73 GB)
+      Raid Devices : 2
+     Total Devices : 2
+       Persistence : Superblock is persistent
+
+       Update Time : Thu Apr 18 11:03:46 2024
+             State : clean
+    Active Devices : 2
+   Working Devices : 2
+    Failed Devices : 0
+     Spare Devices : 0
+
+            Layout : -unknown-
+        Chunk Size : 512K
+
+Consistency Policy : none
+
+              Name : ip-172-31-24-82:0  (local to host ip-172-31-24-82)
+              UUID : e9574118:10d562bf:ed3ca2d9:68ccc3a6
+            Events : 0
+
+    Number   Major   Minor   RaidDevice State
+       0     259        2        0      active sync   /dev/nvme2n1
+       1     259        0        1      active sync   /dev/nvme1n1
+```
+
+[[auth-enable]]
+=== Confirm authentication is enabled
+
+Ensure that xref:manage:security/authentication.adoc[authentication] is set up (or other mitigations are in place). Without SASL authentication enabled, anybody can potentially connect to the Redpanda brokers.
+
+Example:
+```bash
+rpk cluster config get kafka_enable_authorization
+```
+
+Output:
+```bash
+true
+```
+
+[[super-users]]
+=== Confirm that superusers are configured
+
+Ensure that the Admin API is secured, and any users defined in the superusers configuration are appropriately protected with strong credentials.
+
+See also:
+xref:manage:security/authentication.adoc#create-superusers[Create superusers]
+
+[[tls-enabled]]
+=== TLS Enabled 
+
+Ensure that all public interfaces have TLS enabled.
+
+Example:
+```bash
+grep tls /etc/redpanda/redpanda.yaml
+    kafka_api_tls:
+    admin_api_tls:
+    rpc_server_tls:
+        tls:
+        tls:
+    schema_registry_api_tls:
+```
+
+[[redpanda-tuners]]
+=== Run Redpanda tuners
+
+Ensure that tuners have been run on all cluster hosts. xref:reference:rpk/rpk-redpanda/rpk-redpanda-tune.adoc[Redpanda tuners] ensure that the operating system is configured for optimal performance and can have a significant impact on the latency and throughput. In Kubernetes, you may need to run the tuners on the hosts themselves, rather than in containers.
```

Comment:
```suggestion
Ensure that you have run tuners on all cluster hosts. xref:reference:rpk/rpk-redpanda/rpk-redpanda-tune.adoc[Redpanda tuners] ensure that the operating system is configured for optimal performance and can have a significant impact on the latency and throughput. In Kubernetes, you may need to run the tuners on the hosts themselves, rather than in containers.

See also:
- xref:deploy:deployment-option/self-hosted/manual/production/production-deployment.adoc/#tune-the-linux-kernel-for-production[Tune the Linux kernel for production]
- xref:deploy:deployment-option/self-hosted/kubernetes/k-tune-workers/.adoc[Tune Kubernetes Worker Nodes for Production]

```